pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/KubernetesRuntime.java

```diff
@@ -546,6 +546,7 @@ private V1StatefulSet createStatefulSet() {
 
     private Map<String, String> getLabels(Function.FunctionDetails functionDetails) {
         final Map<String, String> labels = new HashMap<>();
+        labels.put("component", "function");
```

Comment:
its not always going to be a function.  It could be a source or a sink

Reply:
Since we do make the distinction in the prometheus metrics exposed in the function instance.  Pulsar sinks stats will be prefixed with "pulsar_sink", sources will be prefixed with "pulsar_source" and functions will be prefixed with "pulsar_function"